### ISSUE TITLE
Rebuild the deployment process & disk images for v1.5

### ISSUE THREAD

**Issue by marcoherbst** [evercam/evercam-gateway#52]

**Comment by scandox** [comment]
What does this mean? Can you write a body for it?

Does it for example overlap with this issue:
https://github.com/evercam/evercam-gateway/issues/29

**Comment by marcoherbst** [comment]
Kinda, I'm just suggesting that there are probably going to be changes required to the "create a new gateway" flow once this Milestone is complete. If I'm wrong, close the issue.

**Comment by scandox** [comment]
Yes actually you're right, since Janus will need to be bundled with it. If it were only code changes to Gateway those don't need a revised process, as the system auto-updates. But it will be a relatively light task. I imagine.
